{
  "contest": 342,
  "handle": "pynschon",
  "risk": "2",
  "title": "Vault incorrectly uses `totalSupply()` to check if `GlobalLendLimit` was reached",
  "issueId": 239,
  "issueUrl": "https://github.com/code-423n4/2024-03-revert-lend-findings/issues/239"
}